doc/sphinx/install.rst

```diff
@@ -238,6 +238,69 @@ To run the backend (still in the virtualenv)::
 The -B option starts the scheduler for periodic tasks, the -l option sets the debug level
 to INFO.
 
+In production you want to run ``celery`` and ``celerybeat`` as a daemon and be controlled by ``systemd``. ``celery`` and ``celerybeat`` are installed in the virtual environment of dissemin, so you have to take care, to use this environment. In particular you should use the same user for dissemin and celery.
+
+You can use the following sample files. Put this into ``/etc/default/celery`` and change ``CELERY_BIN`` path.::
+
+    # See
+    # http://docs.celeryproject.org/en/latest/tutorials/daemonizing.html#available-options
+
+    CELERY_APP="dissemin.celery:app"
+    CELERYD_NODES="dissem"
+    CELERYD_OPTS=""
+    CELERY_BIN="/path/to/venv/env/bin/celery"
+    CELERYD_PID_FILE="/var/run/celery/%n.pid"
+    CELERYD_LOG_FILE="/var/log/celery/%n.log"
+    CELERYD_LOG_LEVEL="INFO"
+
+    CELERYBEAT_SCHEDULE_FILE="/var/run/celery/beat-schedule"
+    CELERYBEAT_PID_FILE="/var/run/celery/beat.pid"
+    CELERYBEAT_LOG_FILE="/var/log/celery/beat.log"
+
+
+For ``celeryd`` systemd service put the following in ``/etc/systemd/system/celery.service`` and change ``WorkingDirectory`` to your dissemin root.::
+
+    [Unit]
+    Description=Celery Service
+    After=network.target
+
+    [Service]
+    Type=forking
+    User=dissemin
+    Group=dissemin
+    EnvironmentFile=-/etc/default/celery
+    WorkingDirectory=/path/to/dissemin/
+    ExecStart=/bin/sh -c '${CELERY_BIN} multi start ${CELERYD_NODES} -A ${CELERY_APP} --pidfile=${CELERYD_PID_FILE} --logfile=${CELERYD_LOG_FILE} --loglevel=${CELERYD_LOG_LEVEL} ${CELERYD_OPTS}'
```

Comment:
Could you add a comment here that you need to call `/bin/sh -c` to be able to actually process the `CELERY_BIN` environment variable, please? (same below for `celeryd`).

I was a bit puzzled at first by this syntax :)